### PR TITLE
FIX: Exported dashboard include response of the queries #1981

### PR DESCRIPTION
**Pull request by FellipeMTX**
Issue solved: #1981 
I created a new ''const'' according to the issue's request and set this const to be the parameter of the download function. I had trouble reproducing the error, but I think this solution should work.
Image of console.log of new const(which will be downloaded):
Query

### CODE REVIEW

File: frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx [hunk 1]
```diff
@@ -66,7 +76,7 @@ function ShareModal({
 					<Button
 						type="primary"
 						onClick={(): void => {
```
Comment (palashgdev) [inline frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx:78]: can make use of closure which will avoid creation of inline function

File: frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx [hunk 2]
```diff
@@ -51,6 +51,16 @@ function ShareModal({
 		}
 	}, [state.error, state.value, t]);
 
+	const cleanedQueryData = selectedData = {...selectedData, widgets: selectedData.widgets?.map((widget) => ({
```
Comment (palashgdev) [inline frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx:54]: can we move this logic to utils ?

**Review by palashgdev — CHANGES_REQUESTED** [review]

File: frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx [hunk 3]
```diff
@@ -66,7 +67,8 @@ function ShareModal({
 					<Button
 						type="primary"
 						onClick={(): void => {
-							downloadObjectAsJson(selectedData, selectedData.title);
+							downloadObjectAsJson(selectedDataCleaned, selectedData.title);
+							console.log(selectedDataCleaned, 'SDC');
```
Comment (palashgdev) [inline frontend/src/container/NewDashboard/DescriptionOfDashboard/ShareModal.tsx:71]: @FellipeMTX console

**Review by palashgdev — CHANGES_REQUESTED** [review]

**Comment by palashgdev** [comment]
Apart from this one PR looks good to me ,Thanks for raising the PR @FellipeMTX :-D

**Review by palashgdev — APPROVED** [review]

**Review by prashant-shahi — APPROVED** [review]
LGTM :shipit:

**Pull request merged**